packages/fx-core/resource/package.nls.json

```diff
@@ -395,6 +398,12 @@
   "core.createProjectQuestion.noPlugin.detail": "Create declarative Copilot only",
   "core.createProjectQuestion.addPlugin.label": "Add plugin",
   "core.createProjectQuestion.addPlugin.detail": "Create declarative Copilot with API plugin",
+  "core.createProjectQuestion.addExistingPlugin.title": "Import from Existing - Upload files",
+  "core.createProjectQuestion.addExistingPlugin.pluginManifest.placeholder": "Select your plugin manifest file",
+  "core.createProjectQuestion.addExistingPlugin.openApiSpec.placeholder": "Select the OpenAPI description used for your API plugin",
+  "core.createProjectQuestion.addPlugin.MissingRequiredProperty": "Invalid plugin manifest. Missing \"%s\"",
+  "core.createProjectQuestion.addPlugin.pluginManifestMissingApiSpec": "Invalid plugin manifest. Make sure the manifest has runtime of \"%s\" type and reference a valid API description document",
```

Comment:
Invalid plugin manifest. Ensure the manifest has a runtime of "%s" and references a valid API description document.